src/Couchbase.Lite.Shared/API/Sync/ReplicatorConfiguration.cs

```diff
@@ -190,8 +184,7 @@ public IList<string> DocumentIDs
         /// <summary>
         /// Extra HTTP headers to send in all requests to the remote target
         /// </summary>
-        [NotNull]
-        public IDictionary<string, string> Headers
+        public IDictionary<string, string?> Headers
```

Comment:
Headers is nullable.

Reply:
I don't like allowing null collections....there is no point and just adds extra null checks.  This was notnull before according to the attribute.